isis/scripts/isisVarInit.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python
+"""
+This program builds shell scripts that define the ISIS environment
+variables that are sourced during conda environment activation and
+removed at deactivation.
+
+If the directories don't exist, they are created.  If their path is
+not specified, they are placed in $ISISROOT.
+"""
+
+import argparse
+import os
+from pathlib import Path
+
+# This is free and unencumbered software released into the public domain.
+#
+# The authors of ISIS do not claim copyright on the contents of this file.
+# For more details about the LICENSE terms and the AUTHORS, you will
+# find files of those names at the top level of this repository.
+#
+# SPDX-License-Identifier: CC0-1.0
+
+
+def mkdir(p: Path):
+    # This just wraps and reports on the directory creation:
```

Comment:
Convert this to a proper doc-string to match the other helper functions.

Reply:
I did.  I also altered this to return a string instead of None.  I don't like functions calling print() (unless I'm debugging).